commit cd81e799b5e176a20c190a002eee28f17633f2d0
Author: Susmit Pal <sushmit@broadcom.com>
Date:   Fri Sep 18 13:30:48 2015 -0700

    CSP962161:QT Snow on 15.2: Pushing workaround to skip format checks'

diff --git a/rockford/middleware/vc5/driver/middleware/khronos/glxx/glxx_server_texture.c b/rockford/middleware/vc5/driver/middleware/khronos/glxx/glxx_server_texture.c
index f138a56..0f4a216 100755
--- a/rockford/middleware/vc5/driver/middleware/khronos/glxx/glxx_server_texture.c
+++ b/rockford/middleware/vc5/driver/middleware/khronos/glxx/glxx_server_texture.c
@@ -1768,15 +1768,6 @@ static void texSubImageX(GLenum target, GLint level, GLint
 
    face = glxx_texture_get_face(target);
 
-   // We must also check for valid combination with the existing internal format
-   GLenum apifmt, typ;
-   gfx_lfmt_to_format_type_maybe(&apifmt, &typ, texture->img[face][level]->api_fmt);
-   if (apifmt != GL_NONE && apifmt != format)
-   {
-      error = GL_INVALID_OPERATION;
-      goto end;
-   }
-
    glxx_texture_subimage(texture, face, level, xoffset, yoffset, zoffset,
                format, type, &(state->pixel_store_state),
                state->bound_buffer[GLXX_BUFTGT_PIXEL_UNPACK].obj, pixels,
